packages/react/src/screens/auth-blocks/email-verify/EmailLinkSent.tsx

```diff
@@ -1,99 +1,60 @@
 import type { EmailVerifyBlock } from '@corbado/shared-ui';
-import { AuthType } from '@corbado/shared-ui';
 import React, { useEffect, useMemo, useRef, useState } from 'react';
 import { Trans, useTranslation } from 'react-i18next';
 
-import { Body, EmailProviderButtons, Header, PrimaryButton } from '../../../components';
+import { PrimaryButton } from '../../../components/ui2/buttons/PrimaryButton';
+import { EmailLinks } from '../../../components/ui2/EmailLinks';
+import { Header } from '../../../components/ui2/typography/Header';
+import { Text } from '../../../components/ui2/typography/Text';
+import { UserInfo } from '../../../components/ui2/UserInfo';
+import { EmailLinkSuccess } from './EmailLinkSuccess';
 
-// we poll for a maximum of 5 minutes (300 * 1000ms = 5min)
-const pollIntervalMs = 1000;
-const pollMaxNumber = 300;
+// we poll for a maximum of 10 minutes (120 * 5000ms = 10min)
+const pollIntervalMs = 5000;
```

Comment:
let's reduce the interval again to 2s pls (right now it can take some time for the old tab to update it's state after verification)